plugins/chat/spec/plugin_helper.rb

```diff
@@ -2,6 +2,8 @@
 
 require "faker"
 
+KEY_MODIFIER = RUBY_PLATFORM =~ /darwin/i ? :meta : :control
```

Comment:
I realise you're just refactoring what was already done... so ✅. But still, using a global constant feels like a code smell. Would this be better?

```
let(:key_modifier){ RUBY_PLATFORM =~ /darwin/i ? :meta : :control }
```
or use `self::SOME_CONSTANT = ` (per [this stackoverflow answer](https://stackoverflow.com/a/6025300/5913559))

Reply:
`self::` is TIL but too much ✨magic✨ for my taste 😄 I'll change it to `let`